.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -230,7 +230,7 @@ jobs:
     needs:
       - dependency-check
       - spotless
-    if: ${{ github.event_name != 'pull_request' || (github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.full_name == github.repository) }}
```

Comment:
We need to leave the JRS panel disabled for dependabot per conversations with @nathanklick

Reply:
Sure I can leave it, but as long as this is a `(required)` check, the PR won't be mergable. How can we configure this in GitHub?

See this entry in the. checks below

```
Node: PR Checks / JRS Panel / Platform SDK (pull_request) Successful in 22m       (Required)
```